### DISCUSSION TITLE
Issue in ADAPTER_EVENTS.ERRORED

### DISCUSSION THREAD

**Discussion by ankitthakur888antr** [Web3Auth/Web3Auth#1134] [Help]
In react application when the user is closing the tab without logging out and again opening in new tab then it is again asking for the permission(for provider change), it's okay when we are clicking confirm but when we are clicking cancel it is not handling anything, directly it gives me the webauth status : "errored" there is no way i can handle it i have tried all the best options. My question is :-

Do i have to reInitialize the web3Auth again? if yes, then Do i have to refresh the window forcefully to reinitialize web3Auth again? or is there another way to reinitialize or other way to solve that problem? 

Or there is anything which iam missing so far?

**Comment by shahbaz17** [comment]
Hey @ankitthakur888antr 

Could you share some information here, like:
* SDK and its version
* Code snippet for initialization and login
* Browser Console screenshots
